common/components/common/projects/EditProjectForm.jsx

```diff
@@ -155,7 +155,6 @@ class EditProjectForm extends React.PureComponent<Props,State> {
           <ImageUploadFormElement form_id="project_thumbnail_location" currentImage={this.state.formFields.project_thumbnail}/>
```

Comment:
The thumbnail upload button looks better as a standard button than the new '+' button, so we'll want to update the css here.